src/libs/actions/IOU.ts

```diff
@@ -448,6 +448,35 @@ type CreateDistanceRequestInformation = {
     policyParams?: RequestMoneyPolicyParams;
 };
 
+type TrackExpenseTransactionParams = {
+    amount: number;
+    currency: string;
+    created: string | undefined;
+    merchant: string;
+    comment: string;
```

Comment:
Can we make these optional and default them to an empty string within the trackExpense function? Looks like most places we pass them as empty strings.